cli/azd/pkg/environment/azdcontext/azdcontext.go

```diff
@@ -89,6 +89,10 @@ var (
 	ErrNoProject = errors.New("no project exists; to create a new project, run `azd init`")
 )
 
+var (
+	ErrNoEnvironment = errors.New("no environment found to get values from")
```

Comment:
We may want to include a helper text like we do with ErrNoProject

Reply:
I don't know what is the expected action that should happen after this.
Should I ask the user to run azd init to create a new project?

As the issue states we say that but it doesn't mention what is expected to happen after that.
https://github.com/Azure/azure-dev/issues/2567#issue-1819518704

Reply:
done